### DISCUSSION TITLE
Δημιουργία ομάδας για το σύνολο της τάξης

### DISCUSSION THREAD

**Discussion by epidrome** [courses-ionio/cscw#168] [Announcements]
### Για να φτιάξετε την ομάδα σας (Προθεσμία τέλος 1ης εβδομάδα, δλδ Κυριακή 19 Φεβρουαρίου)

>> Σε αυτό το μάθημα επιλογής όλη η τάξη θα είναι μια ομάδα. Η δημιουργία της ομάδας και ο συντονισμός θα πρέπει να γίνουν με διαφανείς διαδικασίες και ψηφοφορίες. Ο πρώτος που θα συμπληρώσει το πρώτο σχόλιο παρακάτω θα δημιουργήσει τον αντίστοιχο οργανισμό και σε συνεργασία με τα μέλη θα καταγράψει δημόσια τους κανόνες λειτουργίας. Νέα μέλη θα γίνονται δεκτά μόνο στα σχόλια παρακάτω με **αποκλειστική προθεσμία 19 Φεβρουαρίου.** Μετά την ημερομηνία αυτή νέα μέλη γίνονται δεκτά με μειωμένο βαθμό που φτάνει στην βάση την 5η εβδομάδα, γιατί δεν θα έχουν βοηθήσει στην αρχική οργάνωση της ομάδας.

1. Η [δημιουργία και ο συντονισμός ομάδας](https://epidrome.github.io/teaching/team/) είναι μια δύσκολη τέχνη, για αυτό, εκτός από τις οδηγίες, σας δίνουμε αρκετό χρόνο και σχετικό πλαίσιο (=παράλληλη αποστολή εισαγωγής στις συζητήσεις της βοήθειας, ώστε να γίνει η γνωριμία).
2. Για να δημιουργήσετε μια νέα ομάδα, αφού πρώτα φτιάξετε έναν [νέο λογαριασμό οργανισμού](https://docs.github.com/en/organizations), κάντε ένα νέο σχόλιο αντιγράφοντας τον παρακάτω πίνακα και περιμένετε να απαντήσουν στο σχόλιο σας.
3. Όπως γράφουν και οι αντίστοιχες οδηγίες θα εκτιμηθεί θετικά η δημιουργία ανοικτών ομάδων με διαφανείς κανόνες (π.χ., είσοδος-έξοδος μέλους) και διαδικασίες (που θα ορίσετε εσείς) αντί της δημιουργίας κλειστών ομάδων με προφορικές και κλειστές διαδικασίες, γιατί αυτός είναι ο σύγχρονος (και μελλοντικός) τρόπος (συν)εργασίας.

Η δημιουργία και ο συντονισμός της ομάδας είναι δική σας ευθύνη και θα γίνει σε νέο προφίλ οργανισμού που θα δημιουργήσετε στο γκιτχαμπ και θα γράψετε παρακάτω:

Μετά την δημιουργία της ομάδας αντιγράψτε και συμπληρώστε τον παρακάτω πίνακα στην πρώτη ανάρτηση της ομάδας σας παρακάτω:

```
Όνομα ομάδας: delta
GitHub organisation url: [delta](url)
Νέα μέλη δεκτά: ΝΑΙ/ΟΧΙ

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | 2003001 | gh link |
| 2 | 2003002 | gh link |
| 3 | ... | ... |
```

Παρακαλούμε χρησιμοποιήστε την παρακάτω περιοχή αποκλειστικά για την οργάνωση ομάδων και όχι για απορίες.

**Comment by AlxiKan** [comment]
Όνομα ομάδας: Git Gud or Get Rekt
GitHub organisation url: [AlxiKan](https://github.com/git-gud-or-get-rekt)
Νέα μέλη δεκτά: ΝΑΙ

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | 2016072 | [AlxiKan](https://github.com/AlxiKan) |
| 2 | 2021013 | [nkanagno](https://github.com/nkanagno) |
| 3 | 2021055 | [Axileaszervos](https://github.com/Axileaszervos) |

**Reply by nkanagno** [reply]
Καλησπέρα συνάδελφε, θα μπορούσα να μπω στην ομάδα σου?

**Reply by Axileaszervos** [reply]
Καλησπέρα συνάδελφε, θα μπορούσα να γίνω μέλος της ομάδας σου?

**Reply by epidrome** [reply]
@AlxiKan @nkanagno @Axileaszervos dream team 😍  μην ξεχάσετε να δηλώσετε παρουσία στην φόρμα της διπλανής συζήτησης για την ερχόμενη Δευτ-Τρι.
https://github.com/courses-ionio/cscw/discussions/170

…και συγνώμη για την εκτός θέματος παρέμβαση μου εδώ!
